section4/exercises/classMethods.js

```diff
@@ -28,13 +28,28 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+  changeProtein(protein) {
+    return this.protein = protein;
+  }
+  addTopping(toppings) {
```

Comment:
note that if we're only pushing in one topping at a time, topping would be a better name for this param. However good job on this file overall!